smi/internal/module.go

```diff
@@ -312,6 +321,19 @@ func GetModule(name string) (*Module, error) {
 	return LoadModule(name)
 }
 
+func LoadModuleFromStream(data []byte) (*Module, error) {
```

Comment:
consider naming this `LoadModuleFromReader` and passing in an `io.Reader`

Reply:
done.